addons/mail/static/src/core/public_web/discuss_search.js

```diff
@@ -43,6 +43,10 @@ export class DiscussSearch extends Component {
         return this.props.class;
     }
 
+    get inputContainerExpandedRoundnessClass() {
+        return "rounded-3";
+    }
```

Comment:
This getter is a bit complicated and only exists for a small style tweak. If we do this everywhere in the UI, our components will get way too complex.

It makes sense for internal users to have access to all the discuss features, even on the public page. I’d rather removing those patches and just applying the style directly in the `public_web` bundle. That way we clean up this arbitrary separation, while still keeping the style you wanted 🙂